content/docs/command-reference/metrics/index.md

```diff
@@ -10,24 +10,37 @@ A set of commands to add, manage, collect, and display project metrics:
 ## Synopsis
 
 ```usage
-usage: dvc metrics [-h] [-q | -v] {show,add,modify,remove} ...
+usage: dvc metrics [-h] [-q | -v] {show,add,modify,remove,diff} ...
 
 positional arguments:
-  COMMAND
-    show                Output metric values.
-    add                 Tag file as a metric file.
-    modify              Modify metric file values.
-    remove              Remove files's metric tag.
+  {show,add,modify,remove,diff}
+                        Use `dvc metrics CMD --help` to display command-
+                        specific help.
+    show                Print metrics, with optional formatting.
+    add                 Mark a DVC-tracked file as a metric.
+    modify              Modify metric default formatting.
+    remove              Remove metric mark on a DVC-tracked file.
+    diff                Show changes in metrics between commits
```

Comment:
Also, does it matter that the nav menu order is not the same as the output? If so, which one is right?

![image](https://user-images.githubusercontent.com/1477535/78108691-21b4e700-73b5-11ea-8380-212e7b33378d.png)

Reply:
Cc @shcheklein

Reply:
I don't think it is important. But yes, it would be nice to make them the same.

Reply:
OK, adjusted a little in 774edc74 (for #1110) and opened iterative/dvc/issues/3608.